src/components/UnitCard.js

```diff
@@ -12,27 +16,36 @@ const UpgradeBar = styled.div`
   display: flex;
 `;
 
-const RemoveButton = styled.button`
+const RemoveButton = styled.div`
   float: right;
 `;
 
-const UnitCard = ({ unit, deleteFunction, index }) => {
+const UnitCard = ({ unit, deleteFunction, index, updateFunction }) => {
```

Comment:
If you like trimmed down callback names, `onUpdate` and `onDelete` are coolio conventions